src/style.css

```diff
@@ -1,3 +1,71 @@
+html {
+  font-family: Roboto, sans-serif;
+
+  --link-main-color: #000;
+  --link-active-color: #00acdc;
+}
+
 body {
+  margin: 0 auto;
+}
+
+.header {
+  padding: 0 50px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.header_logo {
+  width: 40px;
+  height: 40px;
+  display: flex;
+  justify-content: center;
+}
+
+.nav_list {
   margin: 0;
+  padding: 0;
+  list-style: none;
+  display: flex;
+  justify-content: space-between;
+}
+
+.nav_item {
+  margin: 0 0 0 20px;
+}
+
+.nav_link {
+  position: relative;
+  display: block;
+  line-height: 60px;
+  text-decoration: none;
+  text-transform: uppercase;
+  font-size: 12px;
+  font-weight: 500px;
```

Comment:
The `font-weight` property does not accept a unit of measurement (px). It should be a numeric value, such as `font-weight: 500;`.